### DISCUSSION TITLE
Antd Carousel autoplay speed

### DISCUSSION THREAD

**Discussion by YasinDemirkol** [ant-design/ant-design#34199] [Q&A] [answered]
Hello!

Is there a way to increase the time between each slide? It goes too fast from slide to slide

**Comment by huntdream** [comment, marked as answer]
Try this [autoplaySpeed](https://react-slick.neostack.com/docs/api/#autoplaySpeed)

**Reply by MansurMahammad** [reply]
This is using React Slick, But we need it in Ant Design

**Reply by huntdream** [reply]
> This is using React Slick, But we need it in Ant Design

Carousel is based on React Slick

**Reply by Alukesh** [reply]
this halped me wery much) thanks

**Reply by yoyo837** [reply]
> Carousel is based on React Slick

It’s a fork repo.
